snmp/datadog_checks/snmp/data/profiles/checkpoint-firewall.yaml

```diff
@@ -0,0 +1,87 @@
+extends:
+  - _base.yaml
+  - _generic-router-if.yaml
+
+sysobjectid: 1.3.6.1.4.1.2620.1.*
+
+metrics:
+    # CPU
+  - MIB: CHECKPOINT-MIB
+    table: multiProcTable
+    symbols:
```

Comment:
While not required, it should be a performance benefit to include the explicit OIDs for our base profiles.  Relatively easy with `ddev meta snmp translate-profile`.

Reply:
Yup, done